.github/workflows/main.yml

```diff
@@ -197,5 +256,5 @@ jobs:
       # and then delete the empty folders
       - run: find release/dist -type d -empty -delete
       - run: ls release/dist
-      - run: pip install tox
-      - run: tox -e cibuild -- upload
+      - run: python release/pins/pip-install-pinned.py
+      - run: python release/cibuild.py upload
```

Comment:
We do not use tox anymore for all jobs that eventually produce release assets. The reason for that is that we want to install all dependencies with their hashes pinned and then install mitmproxy as editable, and that is not (easily) possible with tox.